libindy/Cargo.toml

```diff
@@ -56,6 +56,9 @@ indy-api-types = { path = "./indy-api-types"}
 indy-utils = { path = "./indy-utils"}
 indy-wallet = { path = "./indy-wallet"}
 quote = "=1.0.7"
+variant_count = "*"
+num-traits = "0.2"
+num-derive = "0.2"
```

Comment:
Do we need ALL of these?

Reply:
```variant_count``` - to get CommandIndexes size
```num-traits``` - to get an element of enum by its index
```num-derive`` - to add ```FromPromitive``` derive to CommandIndexes for the previous action.